### DISCUSSION TITLE
ENOENT error when trying to install blitz recipe

### DISCUSSION THREAD

**Discussion by spidertwin2** [blitz-js/blitz#2596] [Q&A] [answered]
`blitz install chakra-ui` gives the output -

```
◢ Cloning GitHub repository for chakra-ui recipe
     Error: ENOENT: no such file or directory, 
     mkdir '/Users/spidertwin/Desktop/myapp/.blitz/recipe-install'
     Code: ENOENT
◤ Cloning GitHub repository for chakra-ui recipe%  
```

running my project on macOS Big Sur 11.4

🐰

**Comment by spidertwin2** [comment, marked as answer]
found a workaround - `git clone` another blitz project that has the recipe installed. Run `yarn` from project directory. And you are set!

🐢

**Comment by flybayer** [comment]
This is a bug. Seems to happen if you run `blitz install` before `blitz dev` or `blitz build`.

**Reply by flybayer** [reply]
I opened #2608
